src/OmniSharp.Host/project.json

```diff
@@ -8,36 +8,35 @@
     "OmniSharp.Stdio": "1.0.0",
     "OmniSharp.Plugins": "1.0.0",
     "OmniSharp.Roslyn": "1.0.0",
-    "Microsoft.AspNetCore.Diagnostics": "1.0.0",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0",
+    "Microsoft.AspNetCore.Diagnostics": "1.1.0",
```

Comment:
we should review these ASP.NET Core dependencies at some point, I don't think all of them are used anymore